client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -159,6 +160,8 @@ function MetadataListEditingDirective() {
         templateUrl: 'scripts/superdesk-authoring/metadata/views/metadata-terms.html',
         link: function(scope) {
 
+            subjectScope = scope;
```

Comment:
We use subjectScope for defining object in variable, so we can use it on service.